docs/tutorials/network-and-buffering-config.md

```diff
@@ -15,6 +15,8 @@ identical:
 ```js
 retryParameters: {
   timeout: 0,       // timeout in ms, after which we abort; 0 means never
+  stallTimeout: 0,  // stall timeout in ms, after which we abort; 0 means never
```

Comment:
Can you set these in the docs to their default values, please?

Reply:
If I understand you correctly, you want the default value in the ShakaPlayer configuration to use 0 for both new configurations, is that correct?

Reply:
No, sorry.  I mean that the docs should show the default values.  You gave non-zero defaults, and I agree with those defaults.

Reply:
Sorry for taking time to respond, Friday was bank holiday in Spain. I fix it now.